lib/measured/rails/validations.rb

```diff
@@ -1,3 +1,6 @@
+require 'active_model'
+require 'active_model/validations'
```

Comment:
I'd prefer to see these in [base.rb](https://github.com/Shopify/measured-rails/blob/master/lib/measured/rails/base.rb), as base is always loaded and loads this file and, so it's not like these will be loaded lazily.

I prefer to be able to look in one place for the required dependencies, and not find them littered at the top of various class files.